synapse/storage/event_federation.py

```diff
@@ -442,6 +442,28 @@ def _get_missing_events(self, txn, room_id, earliest_events, latest_events,
         event_results.reverse()
         return event_results
 
+    @defer.inlineCallbacks
+    def get_forward_events(self, event_ids):
```

Comment:
I find this name a bit confusing - it sounds a bit like forward extremities.

get_successor_events ?